charts/cryostat/tests/service_test.yaml

```diff
@@ -0,0 +1,74 @@
+suite: test service.yaml
+templates:
+  - service.yaml
+
+tests:
+  - it: should create a Service with the correct settings
+    set:
+      core.service.type: ClusterIP
+      core.service.httpPort: 8080
+      authentication.openshift.enabled: false
+    asserts:
+      - hasDocuments:
+          count: 1
+      - equal:
+          path: kind
+          value: Service
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-cryostat
+      - equal:
+          path: spec.type
+          value: ClusterIP
+      - equal:
+          path: spec.ports[0].port
+          value: 8080
+      - equal:
+          path: spec.ports[0].targetPort
+          value: 4180
+      - equal:
+          path: spec.ports[0].protocol
+          value: TCP
+      - equal:
+          path: spec.ports[0].name
+          value: cryostat-http
+      - equal:
+          path: spec.ports[1].port
+          value: 443
+      - equal:
+          path: spec.ports[1].targetPort
+          value: 8443
+      - equal:
+          path: spec.ports[1].protocol
+          value: TCP
+      - equal:
+          path: spec.ports[1].name
+          value: cryostat-https
```

Comment:
Hey @ebaron @andrewazores, I notice here we set the https port even though we don't have TLS setup yet. Is it okay to do so?

Reply:
I guess it's "okay" depending on how the Route or Ingress are configured. If TLS is terminated at the edge rather than passed through then this probably gets effectively ignored anyway. The Route is always created pointing at service port 4180, which is the HTTP port, so this only makes sense when either using no TLS or using edge termination on the Route.

This means right now there might be some ways to get an invalid setup which has halfway-configured TLS and in the end traffic won't actually flow.

Reply:
> This means right now there might be some ways to get an invalid setup which has halfway-configured TLS and in the end traffic won't actually flow.

```
NAME         TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)            AGE     SELECTOR
cryostat     ClusterIP   10.99.43.93   <none>        8181/TCP,443/TCP   3m20s   app.kubernetes.io/instance=cryostat,app.kubernetes.io/name=cryostat
```
I guess it's just odd that the k8s Service shows a 443 port available, but when you port-foward it, the connection is refused. Maybe, we can remove it for now?

Reply:
Forwarding that port would be one case of such an invalid setup :-)

We can remove this port definition for now, since it depends on getting the auth proxies configured to use TLS first before it can actually do anything.

Reply:
Sounds good. #167 :D